INSTALL.rst

```diff
@@ -6,7 +6,7 @@ Installation Requirements
 
 To install Twisted, you need:
 
-- Python 3.5/3.6/3.7.
+- Python 3.6/3.7/3.8/3.9
```

Comment:
Maybe have it like this, so that we don't have to update this file  each time we support or remove a Python version.

```suggestion
- Python (see setup.cfg for supported versions)
```

Reply:
I don't think it's too much of a burden to update it everywhere. It's only once per year or so